src/components/Heading/Heading.stories.mdx

```diff
@@ -62,6 +65,39 @@ A `Heading` component will render a standard HTML `<h>` tag (1-6). The heading's
   </div>
 </Preview>
 
+## DisplaySize Styles
+
+<Story name="DisplaySize Styles" />
+
+<Preview>
+  <div>
+    <Heading
+      id="heading1"
```

Comment:
The page looks good. If you include `<Story>` inside of `<Preview>`, you'll get this display in both the "Canvas" page and the "Docs" page. Right now, the initial "Canvas" page is blank when you click on it. Something like:
```
<Preview>
  <Story name="DisplaySize Styles" />
    <div>
      <Heading
```

Reply:
Ha!  I wondered why things were not showing unnder the Canvas tab.  Now I know.  I made the change.